etc/sensision-tokengen.mc2

```diff
@@ -0,0 +1,63 @@
+//
+//   Copyright 2023  SenX S.A.S.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+
+//
+// This is an envelope to generate sensision tokens.
+//
+
+JSONPRETTY
+
+'sensision' 'applicationName' STORE
+NOW 100 ADDYEARS 'expiryDate' STORE
```

Comment:
Set the issuance and expiry to fixed values so token generation is deterministic for a given value of IDENT.

Explain that the values are chosen so multiple calls to tokengen sensision-tokengen.mc2 lead to the same token.